Engine/UI/Panel/PanelDebug.cpp

```diff
@@ -25,11 +25,12 @@ void PanelDebug::Render()
 		ImGui::Checkbox("Global bounding boxes", &App->debug->show_global_bounding_boxes);
 		ImGui::Checkbox("Camera Frustum", &App->debug->show_camera_frustum);
 		ImGui::Checkbox("QuadTree", &App->debug->show_quadtree);
+		ImGui::Checkbox("AABBTree", &App->debug->show_aabbtree);
 		ImGui::Separator();
 
 		ImGui::Checkbox("Scene window culling", &App->debug->culling_scene_mode);
 		int culling_mode_int = static_cast<int>(App->debug->culling_mode);
-		if (ImGui::Combo("Culling Mode", &culling_mode_int, "None\0Frustum Culling\0QuadTree Culling"))
+		if (ImGui::Combo("Culling Mode", &culling_mode_int, "None\0Frustum Culling\0QuadTree Culling\0OctTree\0AabbTree Culling\0Combined Culling"))
 		{
 			switch (culling_mode_int)
```

Comment:
As a recommendation, this switch statement is redundant. Substitute the switch for this line: 
`App->debug->culling_mode = (ModuleDebug::CullingMode)culling_mode_int;`

Reply:
I think that you are right but it is more readable the way it is now, easier to understand :D